ruby/lib/minitest/queue.rb

```diff
@@ -35,6 +35,31 @@ def message
     end
   end
 
+  class Flaked < Skip
+    attr_reader :failure
+
+    def initialize(failure)
+      super()
+      @failure = failure
+    end
+
+    def result_label
+      "Skipped"
```

Comment:
Does this need to be `Skipped`?

Reply:
@byroot could you weigh in here, I think it's because this is for the other, non-junit reporters

Reply:
IIRC, no. It could be anything, but I can't be 100% sure what reporters do in the wild so I'd rather try to mimick skipped tests as much as possible